src/_gettsim/arbeitslosengeld_2/freibeträge_vermögen.py

```diff
@@ -109,33 +57,17 @@ def maximaler_grundfreibetrag_vermögen(
 
 @policy_function(start_date="2023-01-01")
 def freibetrag_vermögen_in_karenzzeit_bg(
```

Comment:
I am not fan of calling the per-person parameter `schonvermögen` and the aggregated one `freibetrag_vermögen`, unless this is what happens in the law. 

Also, can we rename to `vermögensfreibetrag` everywhere or is a different term used in SGB II ?